README.md

```diff
@@ -31,6 +31,7 @@ You can pass a number of options to the plugin to control various settings.
 | filename | string | 'map' | Name of the file if export only option set to true |
 | hidden | Boolean | false | Set to true if you don't want to display the toolbar. Instead you can create your own buttons or fire print events programtically. You still need to call `addTo(map)` to set the leaflet map context. |
 | hideControlContainer | Boolean | true | Hides the leaflet controls like the zoom buttons and the attribution on the print out. |
+| hideClasses | string | '' | Hides classes on the print out. Use comma separeted classes names as follow : div1, div2, ... |
```

Comment:
Why make it a comma seperated string when you're going to split it into an array later?  I'd suggest an array of strings eg
````
hideClasses: ['div1', 'div2']
````

Reply:
Your are totally right, it's better like with an array.